### PR TITLE
414: Add arms case count to the web views

### PR DESCRIPTION
**Pull request by andrew-js-wright**

### CODE REVIEW

File: app/views/capacity.html [hunk 1]
```diff
@@ -137,7 +137,7 @@ <h3 class="heading-medium">Capacity breakdown</h3>
           <td headers="paroms breakdown">-</td>
           <td headers="fdrs breakdown">-</td>
           <td headers="sdrs breakdown">-</td>
-          <td headers="arms breakdown">-</td>
+          <td headers="arms breakdown">{{ member.armsTotalCases }}</td>
```
Comment (ddebella) [inline app/views/capacity.html]: should this not be rounded too? `{{ member.armsTotalCases | round(1) }}`

File: app/views/capacity.html [hunk 2]
```diff
@@ -158,7 +158,7 @@ <h3 class="heading-medium">Capacity breakdown</h3>
           <td headers="paroms breakdown">-</td>
           <td headers="fdrs breakdown">-</td>
           <td headers="sdrs breakdown">-</td>
-          <td headers="arms breakdown">-</td>
+          <td headers="arms breakdown">{{ member.grades[i].armsTotalCases }}</td>
```
Comment (ddebella) [inline app/views/capacity.html]: Same as above

File: app/views/capacity.html (hunk 1 above)
Comment (andrew-js-wright) [reply app/views/capacity.html]: This is an integer and therefore shouldn't need to be rounded.

File: app/views/capacity.html (hunk 2 above)
Comment (andrew-js-wright) [reply app/views/capacity.html]: Same as above.

**Comment by andrew-js-wright** [comment]
Looking into the failing tests now.

**Pull request merged**